src/server_manager/web_app/gcp_server.ts

```diff
@@ -31,6 +33,39 @@ enum InstallState {
   DELETED
 }
 
+export function getRegionId(zoneId: ZoneId): string {
+  return zoneId.substring(0, zoneId.lastIndexOf('-'));
+}
+
+/** @see https://cloud.google.com/compute/docs/regions-zones */
+export const LOCATION_MAP: {readonly [regionId: string]: GeoId} = {
```

Comment:
Instead of the map, provide a function that returns a GeoLocation given the region id.

Reply:
This map is now private, inside `Zone`.